frontend/components/molecules/NotificationDropdown/index.tsx

```diff
@@ -32,14 +107,16 @@ const NotificationDropdown = () => {
                             </span>
                         </Menu.Item>
                     </div>
-                    <div>
-                        <Menu.Item>{renderNotifications()}</Menu.Item>
+                    <div className={notifications.length ? `max-h-80 overflow-y-auto` : `hidden`}>
+                        {renderNotifications(notifications)}
                     </div>
```

Comment:
It's recommended not to use class or hidden in conditional rendering as it will still display in the website
```suggestion
                    {notifications.length && (<div className='max-h-80 overflow-y-auto'>
                        {renderNotifications(notifications)}
                    </div>)}
```